tests/test_swf.py

```diff
@@ -193,11 +184,10 @@ def test_lowlevel():
     )
 
 
-def test_types():
-
-    need_internet()
+@pytest.mark.needs_internet
+def test_types(tmp_path):
 
-    fname1 = get_remote_file("images/stent.swf", test_dir)
+    fname1 = get_remote_file("images/stent.swf", tmp_path)
```

Comment:
Let's try and not miss any places where a `test_images` fixture might be useful :D